### PR TITLE
Update dao node

### PR DESCRIPTION
**Pull request by chimp1984**
@salokiam @ghubstan Anyone time for a review?

### CODE REVIEW

**Comment by ghubstan** [comment]
> @salokiam @ghubstan Anyone time for a review?

I will review it later today and/or Wed.

File: daonode/src/main/java/bisq/daonode/dto/BondedReputationDto.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+@Getter
+@Slf4j
+@Schema(title = "BondedReputation")
+public class BondedReputationDto {
+    private long amount;
+    private long time;
+    private String hash;
+    private int blockHeight;
+    private int lockTime;
+
```
Comment (ghubstan) [inline daonode/src/main/java/bisq/daonode/dto/BondedReputationDto.java:40]: Should the fields above be `final`?

Comment (chimp1984) [reply daonode/src/main/java/bisq/daonode/dto/BondedReputationDto.java:40]: Not sure if the json conversion uses setters behind the sceen...

Comment (ghubstan) [reply daonode/src/main/java/bisq/daonode/dto/BondedReputationDto.java:40]: I was not sure either, but this DTO's fields are final:  https://github.com/chimp1984/bisq/blob/update_dao_node/daonode/src/main/java/bisq/daonode/dto/ProofOfBurnDto.java

Comment (chimp1984) [reply daonode/src/main/java/bisq/daonode/dto/BondedReputationDto.java:40]: Ah ok, will change then...

File: daonode/src/main/java/bisq/daonode/endpoints/SignedWitnessApi.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    )
+    @GET
+    @Path("get-date/{hash}")
+    public Long getDate(@Parameter(description = DESC_HASH)
+                        @PathParam("hash")
+                        String hash) {
+        AccountAgeWitnessService accountAgeWitnessService = checkNotNull(serviceNode.getAccountAgeWitnessService());
+        return accountAgeWitnessService.getWitnessByHashAsHex(hash)
+                .map(accountAgeWitnessService::getWitnessSignDate)
+                .orElse(-1L);
```
Comment (ghubstan) [inline daonode/src/main/java/bisq/daonode/endpoints/SignedWitnessApi.java:75]: This enpoint's respnose header content-type = "application/json", but the response content-type can be "text/plain", i.e, -1 or a date.  A requester is seeing a "application/json" header in the response, but is getting a "text/plain" value instead.

Should these "text/plain" values be wrapped in json in the response?

Comment (ghubstan) [reply daonode/src/main/java/bisq/daonode/endpoints/SignedWitnessApi.java:75]: Maybe it does not matter, but my main question is "is it OK to send a raw long in the response when its content-type header is "application/json"?

File: daonode/src/main/java/bisq/daonode/endpoints/AccountAgeApi.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                    schema = @Schema(allOf = ProofOfBurnDto.class))}
+    )
+    @GET
+    @Path("get-date/{hash}")
+    public Long getDate(@Parameter(description = DESC_HASH)
+                        @PathParam("hash")
+                        String hash) {
+        return checkNotNull(serviceNode.getAccountAgeWitnessService()).getWitnessByHashAsHex(hash)
+                .map(AccountAgeWitness::getDate)
+                .orElse(-1L);
```
Comment (ghubstan) [inline daonode/src/main/java/bisq/daonode/endpoints/AccountAgeApi.java:74]: I have the same question about the the HTTP response returning a raw Long value (not json wrapped) as I did in this  https://github.com/bisq-network/bisq/pull/6306#discussion_r937838662

File: daonode/src/main/java/bisq/daonode/endpoints/SignedWitnessApi.java (hunk 2 above)
Comment (ghubstan) [reply daonode/src/main/java/bisq/daonode/endpoints/SignedWitnessApi.java:75]: @salokiam, do you have an opinion about this?

Comment (chimp1984) [reply daonode/src/main/java/bisq/daonode/endpoints/SignedWitnessApi.java:75]: OK change to text_plain

**Review by ghubstan — APPROVED** [review]
Tested ACK

**Review by ripcurlx — APPROVED** [review]
utACK - based on https://github.com/bisq-network/bisq/pull/6306#pullrequestreview-1062421051

**Pull request merged**